src/hooks/walletHooks.ts

```diff
@@ -0,0 +1,92 @@
+import { useState, useEffect } from 'react';
```

Comment:
We had deleted the `walletHooks.ts` component completely. Did you update your branch?
Do you wanna restore this component? since it's not used anywhere.

Reply:
@mateodaza JFYI

Reply:
I was looking at it, and I don't need to. I removed the file.